AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/CallActivity.java

```diff
@@ -138,10 +140,7 @@ protected void onCreate(final Bundle savedInstanceState) {
     private void initializeCallStateLiveData() {
         final Observer<CallState> observerCallState = callState -> {
             Log.d(LOG_TAG, "CallActivity initializeCallStateLiveData: " + callState.toString());
-            if (callState == CallState.IN_LOBBY) {
-                hideCallActivityProgressBar();
-                showInLobbyWaitingOverlay();
-            } else if (callState == CallState.CONNECTED) {
+            if (callState == CallState.CONNECTED) {
                 hideCallActivityProgressBar();
                 hideInLobbyWaitingOverlay();
```

Comment:
Now that `hideCallActivityProgressBar()` and `hideInLobbyWaitingOverlay()` run after the call is established, i wonder if it will be entirely redundant to include it in the `CONNECTED` stated.

Reply:
addressed and removed hideCallActivityProgressBar from CONNECTED case.